defaults/main.yml

```diff
@@ -0,0 +1,17 @@
+# Service accounts
+monitoring_sa: monitoring
+autobot_jenkins_sa: autobot-jenkins
+koji_sa: koji
+ansible_tower_sa: ansible-tower
+
+list_sa:
+  - "{{ monitoring_sa }}"
+  - "{{ autobot_jenkins_sa }}"
+  - "{{ koji_sa }}"
+  - "{{ ansible_tower_sa }}"
+
+list_of_dict_sa_with_roles:
+  - { sa_name: "{{ monitoring_sa }}", role: 'view' }
```

Comment:
same as with previous comment, if you want defaults just leave here some example_sa with view role

Reply:
Fixed